dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -542,3 +542,23 @@
         # You should configure this property if the maximum open files are limited on your system, or if you have thousands of files in your RocksDB state as there is a memory overhead to keeping all of them open, and setting maxOpenFiles will bound that.
         # This setting can also be set using the environment variable ZEEBE_BROKER_EXPERIMENTAL_ROCKSDB_MAX_OPEN_FILES
         # maxOpenFiles: -1
+
+        # Configures the maximum number of simultaneous write buffers/memtables RocksDB will have in memory. Normally about 2/3s of the memoryLimit
+        # is used by the write buffers, and this is shared equally by each write buffers. This means the higher maxWriteBufferNumber is, the less
+        # memory is available for each. This means you will flush less data at once, but may flush more often.
```

Comment:
```suggestion
        # memory is available for each. This means you will flush less data at once, but may flush more often.
        # This setting can also be set using the environment variable ZEEBE_BROKER_EXPERIMENTAL_ROCKSDB_MAX_WRITE_BUFFER_NUMBER
```

Reply:
I think it should be `ZEEBE_BROKER_EXPERIMENTAL_ROCKSDB_MAXWRITEBUFFERNUMBER`, but you're right - I forgot we add this to every config.

Reply:
then all others are wrong? :thinking:

Reply:
I think so, except for the column family options one since those are parsed differently for RocksDB.

Reply:
I'll update them